api/Navigator.json

```
@@ -53,11 +53,11 @@
           "support": {
             "chrome": {
               "version_added": false,
-              "notes": "Available on all platforms behind a flag, but currently only works on desktop in an <a href='https://webvr.info/get-chrome/'>experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked)."
+              "notes": "Available on all platforms behind a flag, but currently only works on desktop in an <a href='https://webvr.info/get-chrome/'>experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked). WebVR support is no longer available in Chrome 80 and later."
             },
             "chrome_android": {
               "version_added": true,
-              "notes": "Currently supported only by Google Daydream."
+              "notes": "Only supported only by Google Daydream. WebVR support is no longer available in Chrome 80 and later."
```

Instead of the Chrome 80 note, it'd be better to represent this as `"version_removed"`. Also, there's an "only" sandwich

```suggestion
              "version_removed": "80",
              "notes": "Supported only by Google Daydream."
```